samples/openthread/coap_client/sample.yaml

```diff
@@ -1,26 +1,46 @@
 sample:
   description: Test CoAP communication in Thread network
   name: OpenThreadThread CoAP Client
+
 tests:
-  sample.openthread.coap_client:
+  sample.openthread.coap_client.ftd:
     build_only: true
-    extra_args: SNIPPET=ci
+    tags: ci_build
+    platform_allow: >
+      nrf52840dk_nrf52840
+      nrf5340dk_nrf5340_cpuapp
+      nrf5340dk_nrf5340_cpuapp_ns
+    extra_args: >
+      SNIPPET="ci;logging"
     integration_platforms:
+      - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
       - nrf5340dk_nrf5340_cpuapp_ns
-      - nrf52840dk_nrf52840
-      - nrf21540dk_nrf52840
```

Comment:
should `nrf21540dk_nrf52840`  target be restored for coap_client samples as well?